.ci/validation.sh

```diff
@@ -410,7 +410,7 @@ spotbugs-and-pmd)
   mvn -e --no-transfer-progress clean test-compile pmd:check spotbugs:check
   cd .ci-temp/spotbugs-and-pmd
   grep "Processing_Errors" "$CHECKSTYLE_DIR/target/site/pmd.html" | cat > errors.log
-  RESULT=$(cat errors.log | wc -l)
+  RESULT=$(< errors.log wc -l)
```

Comment:
This is confusing with redirection in subshell, IMO

Reply:
I could have written it as ``wc -l > errors.log`` but this way of writing causes error in ``pitest.sh`` as in the image below:

![image](https://user-images.githubusercontent.com/90137881/225095159-e034d142-1080-435e-8f00-dbd0f1500497.png)

![image](https://user-images.githubusercontent.com/90137881/225095278-5e0d35fb-2d16-46f8-826a-0f29318fad30.png)

Reply:
@nrmancuso Done. (permanent suppression created).